server/src/main/java/org/elasticsearch/index/seqno/RetentionLease.java

```diff
@@ -23,7 +23,7 @@
  * A "shard history retention lease" (or "retention lease" for short) is conceptually a marker containing a retaining sequence number such
  * that all operations with sequence number at least that retaining sequence number will be retained during merge operations (which could
  * otherwise merge away operations that have been soft deleted). Each retention lease contains a unique identifier, the retaining sequence
- * number, and the source of the retention lease (e.g., "ccr").
+ * number, the timestamp of when the lease was created or renewed, and the source of the retention lease (e.g., "ccr").
```

Comment:
nit: This class can be final.

Reply:
I pushed I pushed a6ae2c53d817b2b7ef92644ccddf0937833d8ef1.